docs/DafnyRef/Modules.md

```diff
@@ -275,14 +275,14 @@ module Helpers {
   }
 }
 module Mod {
-  import opened Helpers
+  import opened H = Helpers
   function addOne(n: nat): nat {
```

Comment:
```suggestion
  function addOne(n: nat): nat {
  // incorrect implementation
```

Reply:
The function is not incorrect, though may be in practice it would be confusing.